docs/source/basic/example.rst

```diff
@@ -0,0 +1,76 @@
+Code Example
+============
+
+The following example shows a small hello word example written with ``alpaka`` that can be run on different processors.
+
+.. literalinclude:: ../../../example/helloWorld/src/helloWorld.cpp
+   :language: C++
+   :caption: helloWorld.cpp
+
+Use alpaka in your project
+++++++++++++++++++++++++++
+
+``alpaka`` supports ``CMake`` to integrate it into your own project. There are two possible methods.
```

Comment:
```suggestion
We recommend to use ``CMake`` for integrating ``alpaka``` into your own project. There are two possible methods.
```

Also, what do the \` do in *.rst files? Is it the equivalent to this: `alpaka`?

Reply:
Yes, it is equivalent. Single quotes makes the text italic and double quotes highlight the text (orange text in white box).

Reply:
Okay. Is there a reason you want to write `CMake` and `alpaka` instead of CMake and alpaka?

Reply:
Not really. Some projects highlights the own name and names of other important project and some not. It was only my habit. I think, I will change it, because it isn't done somewhere else in the doc.